core/dom/range.js

```diff
@@ -2818,6 +2818,107 @@ CKEDITOR.dom.range = function( root ) {
 			reference.remove();
 		},
 
+		/**
+		 * Returns an array of {@link CKEDITOR.dom.rect} elements that are represented as rectangles which are covered by ranges.
+		 * For each DOM element within the selection range there is only one rectangle.
+		 * Rectangles represent the area of the screen occupied by the elements contained within the range.
+		 *
+		 * In the following example:
+		 *
+		 *        <p><span>first { span</span><span> second text</span></p>
```

Comment:
Use tabs for indentation instead of spaces.

Reply:
Use triple backticks for code samples, see https://github.com/ckeditor/ckeditor-docs/pull/131

Reply:
Also mark the code listing language for proper syntax highlighting.